PRS10001-11000/PRS10255Yasimma.xml

```diff
@@ -51,23 +51,36 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: place</change>
+         <change when="2019-12-18" who="DE">Added information, relations, and bibliography.</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="2">
-               <persName>
-                  
-                  Yäšimmäbet Ali
-               </persName>
+              <persName xml:id="n1" xml:lang="gez">የሺመቤት፡ አሊ፡</persName>
+               <persName corresp="#n1" xml:lang="gez" type="normalized">Yašimmabet ʾAli</persName>
+               <birth notBefore="1864" notAfter="1866">She was a daughter of <persName ref="PRS9833Walatta"></persName> and
+               <persName ref="PRS1683AliAbba"></persName>.
+             </birth>
+               <floruit>In <date>1894</date> at the age of 11 she married <persName ref="PRS6567Makwanne"></persName>
```

Comment:
To me it seems a comma after 1894 or 11 (should be eleven?) would be nice.

Reply:
I added a comma. As far as I could check small numbers ranging from one to ten should generally be spelled out. Larger numbers (i.e., above ten) are written as numerals, unless they are at the verz beginning of a sentence.